code/game/machinery/_machines_base/machinery_components.dm

```diff
@@ -219,6 +219,8 @@ GLOBAL_LIST_INIT(machine_path_to_circuit_type, cache_circuits_by_build_path())
 /obj/machinery/proc/components_are_accessible(var/path)
 	if(ispath(path, /obj/item/stock_parts/access_lock))
 		return TRUE
+	if(ispath(path, /obj/item/stock_parts/circuitboard/airlock_electronics))
+		return TRUE
```

Comment:
Please not on the base `/machinery`. Also, what does this even do? They don't have forwarded interactions with panel closed.

EDIT: I see why now, but am not convinced you should be doing this. I'd just check access in `cannot_transition_to` or within the attackby code.

Reply:
Moved to door/ one